competencies/roles/TechLeaderLevelMapping.md

```diff
@@ -0,0 +1,7 @@
+                                                                                                                                                                                                
+| Title/Job                             |  Industry Comparison       | Key Competencies                                                                                | Other Competencies              |     Notes  |
+| ------------------------------------- |:-------------------------- |:-----------------------------------------------------------------------------------------------:| -------------------------------:| ----------:|
+| Team Lead                             |  L2 first non-IC role      | Proficient: "Releasing Value"  and "Communication"                                              | 50%  proficient or above prof   |  Lateral to SWDev IV |
+| Eng Manager                           |  L3 typically 2 - 5 years  | Proficient: "Developing Team" and "Facilitation"                                                | 75% proficient; some authority  |  | 
+| Senior Eng Manager / Head of          |  L4 typically < 8 years    | Proficient or Authority "Communication" and "Tech Strategy, Alignment, Directional Leadership"  | 50% proficient; 50% authority   |  | 
```

Comment:
Proficiency in "Communication" is implied via the earlier Team Lead role.
Also, "proficient or authority" translates to "proficient" to me (i.e. "at least proficient").  Is that the intention?

Reply:
I was aiming to connote "proficiency is required" and "authority preferred". I will just make it specific one way or another.

Reply:
made it "authority preferred"